pkg/flowaggregator/flowaggregator.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flowaggregator
+
+import (
+	"net"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware/go-ipfix/pkg/collector"
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/exporter"
+	"github.com/vmware/go-ipfix/pkg/registry"
+)
+
+type flowAggregator struct {
+	flowCollectorAddr net.Addr
+	exportInterval    time.Duration
+}
+
+func InitFlowAggregator(flowCollectorAddr net.Addr, exportInterval time.Duration) *flowAggregator {
+	registry.LoadRegistry()
+	return &flowAggregator{
+		flowCollectorAddr,
+		exportInterval,
+	}
+}
+
+func (fae *flowAggregator) Run(stopCh <-chan struct{}) {
+	// Go-ipfix TCP collector doesn't work during my last test, fixed as udp temporarily.
+	collectAddr, _ := net.ResolveUDPAddr("udp", "0.0.0.0:4739")
+	cp, _ := collector.InitCollectingProcess(collectAddr, 1024, 0)
+
+	ep, err := exporter.InitExportingProcess(fae.flowCollectorAddr, 1, 0)
+	if err != nil {
+		klog.Fatalf("Got error when connecting to %s", fae.flowCollectorAddr.String())
+	}
+
+	exportTicker := time.NewTicker(fae.exportInterval)
+	defer exportTicker.Stop()
+	sentMsgNum := 0
+	go cp.Start()
+	for {
```

Comment:
you may want to review whether all the log messages in this loop are necessary or have the correct verbosity?